### PR TITLE
fix: create Pick List from multiple Sales Orders

### PR DESCRIPTION
**Pull request by Dharmraj-48**
Pick List For Sales Order List
picklist

### CODE REVIEW

**Comment by vjFaLk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Clones added
============
- erpnext/selling/doctype/sales_order/sales_order_list.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DigiThinkIT/erpnext/pullRequest?prid=5165835&bid=16959687)

File: erpnext/selling/doctype/sales_order/sales_order_list.js [hunk 1]
```diff
@@ -47,6 +47,41 @@ frappe.listview_settings['Sales Order'] = {
...
+	},
+
+	onload: function(doclist) {
+		const action = () => {
+			const selected_docs = doclist.get_checked_items();
+			const docnames = doclist.get_checked_items(true);
+
+			if (selected_docs.length > 0) {
+				for (let doc of selected_docs) {
+					if (!doc.docstatus) {
```
Comment (Alchez) [inline erpnext/selling/doctype/sales_order/sales_order_list.js:59]: Also check if the order is already delivered

**Comment by Alchez** [comment]
Made upstream to [hotfix](https://github.com/frappe/erpnext/pull/20980) and [develop](https://github.com/frappe/erpnext/pull/20966).

**Comment by Alchez** [comment]
Closing this since Frappe is not ready to take this feature, we will make this as a customization

**Pull request closed without merging**